zokrates_core/src/typed_absy/integer.rs

```diff
@@ -518,8 +518,10 @@ impl<'ast, T: Field> ArrayExpression<'ast, T> {
                     // try to convert the repeated element to the target type
```

Comment:
is this comment outdated now? should be the inferred type, not the target type I assume

Reply:
Good catch, the right word here is align, not convert: aligning `int[42]` to `field[N]` yields `field[42]`